oat-sa/extension-tao-itemqti-pci
# Audio PCI partial processing

**Changes:**
- reduced chunk size for Media Recorder provider
- async destroy callback added for audio PCI
- partial encoded file generation added for audio PCI

These changes are depending on: https://github.com/oat-sa/extension-tao-itemqti/pull/1201

**To test:**
Create a new delivery which contains an audio PCI and test the following cases:
- Start recording then stop it
- Start recording then wait until the interaction time limit reached
- Start recording then wait until the test time limit reached
- Start recording then navigate to the next item (next button / skip button)

It should be tested with both compressed and uncompressed audio PCI.
Expected result: the recorded audio should appear in the result storage.
